### PR TITLE
YLP-1770 team list is not refreshed

### PR DESCRIPTION
**Pull request by Xwdrblue**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

File: packages/yourloops/components/dialogs/join-team/privacy-policy-confirm.tsx [hunk 1]
```diff
@@ -53,7 +53,6 @@ export const PrivacyPolicyConfirm = (props: ConfirmTeamProps): JSX.Element => {
   const { onCompleteStep, team, onClickCancel, inProgress } = props
   const [policyChecked, setPolicyChecked] = useState<boolean>(false)
   const theme = useTheme()
-
```
Comment (TimCanu) [inline packages/yourloops/components/dialogs/join-team/privacy-policy-confirm.tsx:56]: **❔ question:**  You really don't like to aerate your code, do you?‏

File: packages/yourloops/pages/notifications/notifications-list.tsx [hunk 2]
```diff
@@ -71,24 +71,25 @@ const NotificationsPage: FunctionComponent<PropsWithChildren> = () => {
...
-  const notifications = notificationsHook.receivedInvitations
-
   const handleSwitchRoleOpen = (): void => {
     metrics.send('switch_account', 'display_switch_notification')
     setSwitchRoleOpen(true)
   }
   const handleSwitchRoleCancel = (): void => {
     setSwitchRoleOpen(false)
   }
+  const refreshNotificationHook = async (): Promise<void> => {
```
Comment (TimCanu) [inline packages/yourloops/pages/notifications/notifications-list.tsx:90]: **🎯 suggestion (non-blocking):** ‏
```suggestion
  const refreshReceivedInvitations = async (): Promise<void> => {
```

File: packages/yourloops/pages/notifications/notifications-list.tsx [hunk 3]
```diff
@@ -106,6 +107,7 @@ const NotificationsPage: FunctionComponent<PropsWithChildren> = () => {
                     notification={notification}
                     userRole={user.role}
                     onHelp={handleSwitchRoleOpen}
+                    refreshNotificationHook={refreshNotificationHook}
```
Comment (TimCanu) [inline packages/yourloops/pages/notifications/notifications-list.tsx:110]: **🎯 suggestion (non-blocking):** ‏
```suggestion
                    refreshReceivedInvitations={refreshReceivedInvitations}
```

Comment (TimCanu) [reply packages/yourloops/pages/notifications/notifications-list.tsx:110]: **❔ question:** ‏So passing this function as a prop does not create a memory leak. But directly calling `await notificationsHook.refreshReceivedInvitations()` from the child component would? Am I correct?

Comment (Xwdrblue) [reply packages/yourloops/pages/notifications/notifications-list.tsx:110]: yes it is

Comment (TimCanu) [reply packages/yourloops/pages/notifications/notifications-list.tsx:110]: okay :+1:

**Pull request merged**